Package.swift

```diff
@@ -7,7 +7,7 @@ let package = Package(
     name: "StreamChat",
     defaultLocalization: "en",
     platforms: [
-        .iOS(.v12), .macOS(.v10_15)
+        .iOS(.v13), .macOS(.v11)
```

Comment:
- We need to update the README.me iOS bagde to iOS 13+ (it is still on 11 actually)
- We also need to update the Documentation.docc which we say iOS 12+

Reply:
done